addons/youtube-fullscreen/userscript.js

```diff
@@ -1,5 +1,12 @@
 export default async function ({ addon, console }) {
   const player = await addon.tab.waitForElement(".youtube-player");
-  player.setAttribute("allowfullscreen", true);
-  player.src += ""; // reload video
+
+  function setFullscreenEnabled(enabled) {
+    if (enabled) player.setAttribute("allowfullscreen", true);
+    else player.removeAttribute("allowfullscreen");
+    player.src += ""; // reload video
+  }
+  setFullscreenEnabled(true);
+  addon.self.addEventListener("disabled", () => setFullscreenEnabled(false));
+  addon.self.addEventListener("reenabled", () => setFullscreenEnabled(true));
```

Comment:
Is it reasonable to reload the YouTube iframe dynamically? IMO this addon should have never had dynamicEnable, nor it should also get dynamicDisable

Reply:
I'll remove `dynamicEnable` and `dynamicDisable` from this addon.